.pdm-python

```diff
@@ -0,0 +1 @@
+/Users/blairl/Documents/Dev/subcell-pipeline/.venv/bin/python
```

Comment:
We shouldn't commit this file -- probably add to the `.gitignore`?

Reply:
whoops